### PR TITLE
Dynamic kubelet config is removed

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
The feature was removed from the kubelet

/sig node
KEP: https://github.com/kubernetes/enhancements/issues/281

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 1]
```diff
@@ -8,383 +8,37 @@ min-kubernetes-server-version: v1.11
...
-  reference.
+In order to migrate off the Dynamic Kubelet Configuration feature, the
+alternative mechanism should be used to distribute kubelet configuration files.
+In order to apply configuration, config file must be updated and kubelet restarted.
+See the [Set Kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file/)
+for information.
 
+Please note, the `DynamicKubeletConfig` feature gate cannot be set on kubelet
+any longer as it has no effect. For two more versions (till 1.26) the feature gate
+`DynamicKubeletConfig` will keep working on control plane to support older Nodes
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:43]: it is removed from kubelet so whole new version 1.24 will not have this feature. However you can still use it with older agents, we kept control plane functionality to work with those agents

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:43]: Still I don't quite understand this. Based on the following statement, I'd assume the the feature gate is still available on `kube-apiserver` and/or the `kube-controller-manager`. I would not call that a removal of the dynamic kubelet config figure. It is partial removal.

> For two more versions (till 1.26) the feature gate`DynamicKubeletConfig` will keep working on control plane to support older Nodes.

On some other docs, people may want to treat kubelet as part of the control plane, deployed as a node agent.

Please consider revise the sentences.

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 2]
```diff
@@ -8,383 +8,37 @@ min-kubernetes-server-version: v1.11
...
 The [Dynamic Kubelet Configuration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/281-dynamic-kubelet-configuration)
-feature is deprecated and should not be used.
+feature is deprecated in 1.22 and removed in 1.24.
 Please switch to alternative means distributing configuration to the Nodes of your cluster.
 {{< /caution >}}
 
-
 [Dynamic Kubelet Configuration](https://github.com/kubernetes/enhancements/issues/281)
-allows you to change the configuration of each
+were allowing you to change the configuration of each
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:20]:
```suggestion
allowed you to change the configuration of each
```

File: content/en/blog/_posts/2018-07-11-dynamic-kubelet-configuration.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ date: 2018-07-11
 
 **Author**: Michael Taufen (Google)
 
+**Editor’s note: Since this article publication, the feature was removed in the version 1.24.**
```
Comment (divya-mohan0209) [inline content/en/blog/_posts/2018-07-11-dynamic-kubelet-configuration.md:9]:
```suggestion
**Editor’s note: The feature has been removed in the version 1.24 after deprecation in 1.22.**
```

**Review by divya-mohan0209 — COMMENTED** [review]
Grammatical nits

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:43]: the key point here is that when you install 1.24 version of kubernetes, this feature doesn't work.

How would you suggest to reword the sentence?

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:43]: How about:

```
Please note, the `DynamicKubeletConfig` feature gate cannot be set on a kubelet
starting v1.24 as it has no effect. However, the feature gate is not removed from
the API server or the controller manager before v1.26. This is designed for the
control plane to support nodes with older versions of kubelets and for satisfying
the [Kubernetes version skew policy](/releases/version-skew-policy/).
```

[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: great, applied verbatim. I thought you are asking for a bigger change, thank you for suggestion!

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 4]
```diff
@@ -12,379 +12,34 @@ min-kubernetes-server-version: v1.11
...
-{{< /table >}}
+Please find documentation on this feature in [earlier versions of documentation](https://v1-23.docs.kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/).
 
+## Migrating from using Dynamic Kubelet Configuration
 
-## {{% heading "whatsnext" %}}
+There is no recommended replacement for this feature that works generically
+across various Kubernetes distributions. If you are using managed Kubernetes
+version, please consult with the vendor hosting Kubernetes for the best
+practices for customizing your Kubernetes. If you are using KubeAdm, refer to
```
Comment (sgaist) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:32]: Small typo: kubeadm

Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md:32]: @SergeyKanzhelev

**Comment by reylejano** [comment]
Confirmed with SIG Node that the reconfigure kubelet page can be removed in the #sig-node slack channel in [this thread](https://kubernetes.slack.com/archives/C0BP8PW9G/p1647531446062829)

/lgtm

**Pull request merged**